docs/topics/wildfly-swarm/docs/reference/fractions/hystrix.adoc

```diff
@@ -0,0 +1,120 @@
+
+swarm.hystrix.collapser.default.maxRequestsInBatch:: 
+The maximum number of requests allowed in a batch before this triggers a batch execution
+
+swarm.hystrix.collapser.default.requestCache.enabled:: 
+Indicates whether request caching is enabled for HystrixCollapser.execute() and HystrixCollapser.queue() invocations
+
+swarm.hystrix.collapser.default.timerDelayInMilliseconds:: 
+The number of milliseconds after the creation of the batch that its execution is triggered
+
+swarm.hystrix.command.default.circuitBreaker.enabled:: 
+Determines whether a circuit breaker will be used to track health and to short-circuit requests if it trips
+
+swarm.hystrix.command.default.circuitBreaker.errorThresholdPercentage:: 
+The error percentage at or above which the circuit should trip open and start short-circuiting requests to fallback logic
+
+swarm.hystrix.command.default.circuitBreaker.forceClosed:: 
+If true, forces the circuit breaker into a closed state in which it will allow requests regardless of the error percentage
+
+swarm.hystrix.command.default.circuitBreaker.forceOpen:: 
+If true, forces the circuit breaker into an open (tripped) state in which it will reject all requests
+
+swarm.hystrix.command.default.circuitBreaker.requestVolumeThreshold:: 
+The minimum number of requests in a rolling window that will trip the circuit
+
+swarm.hystrix.command.default.circuitBreaker.sleepWindowInMilliseconds:: 
+The amount of time, after tripping the circuit, to reject requests before allowing attempts again to determine if the circuit should again be closed
+
+swarm.hystrix.command.default.execution.isolation.semaphore.maxConcurrentRequests:: 
+The maximum number of requests allowed to a HystrixCommand.run() method when you are using ExecutionIsolationStrategy.SEMAPHORE
+
+swarm.hystrix.command.default.execution.isolation.strategy:: 
+Isolation strategy (THREAD or SEMAPHORE)
+
+swarm.hystrix.command.default.execution.isolation.thread.interruptOnCancel:: 
+Indicates whether the HystrixCommand.run() execution should be interrupted when a cancellation occurs
+
+swarm.hystrix.command.default.execution.isolation.thread.interruptOnTimeout:: 
+Indicates whether the HystrixCommand.run() execution should be interrupted when a timeout occurs
+
+swarm.hystrix.command.default.execution.isolation.thread.timeoutInMilliseconds:: 
+The time in milliseconds after which the caller will observe a timeout and walk away from the command execution
+
+swarm.hystrix.command.default.execution.timeout.enabled:: 
+Indicates whether the HystrixCommand.run() execution should have a timeout
+
+swarm.hystrix.command.default.fallback.enabled:: 
+Determines whether a call to HystrixCommand.getFallback() will be attempted when failure or rejection occurs
+
+swarm.hystrix.command.default.fallback.isolation.semaphore.maxConcurrentRequests:: 
+The maximum number of requests allowed to a HystrixCommand.getFallback() method when you are using ExecutionIsolationStrategy.SEMAPHORE
```

Comment:
values and method calls should be formatted with backticks

Reply:
Note to self: not easily fixable in our sources.